smurfs/src/components/Smurf.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import  image  from '../images/smurf.jpg'
+
+
+const Smurf = props => {
+    console.log('these are the props passed in smurf', props)
+
+    if(!props) return <h2>Loadings Smurfs</h2>
+
+    return (
+        <div className='smurf-container'>
+            <p className='name'>{props.name}</p>
+            <img width='150px' src={image} alt='smurf lawn knome'/>
+            <p><strong>Age: </strong>{props.age}</p> 
```

Comment:
I usually try to stay away from linline styling. Styled components is my go to but using an external style sheet would help other devs jump in your project